ouroboros-network-framework/ouroboros-network-framework.cabal

```diff
@@ -60,42 +60,41 @@ library
   build-depends:       base            >=4.12 && <4.13
                      , async           >=2.1  && <2.3
                      , bytestring      >=0.10 && <0.11
+                     , cborg           >=0.2.1 && <0.3
                      , containers      >=0.5  && <0.7
+                     , dns                       < 4.0
+                     , iproute         >= 1.7.0 && < 1.8
                      , mtl
                      , stm
                      , text
                      , time
 
-                     , cborg           >=0.2.1 && <0.3
+                     , cardano-prelude
+                     , contra-tracer
+
                      , io-sim-classes  >= 0.1 && < 0.2
-                     , typed-protocols >= 0.1 && < 0.2
                      , network         >= 3.1 && < 3.2
                      , network-mux     >= 0.1 && < 0.2
-                     , contra-tracer
+                     , typed-protocols >= 0.1 && < 0.2
                      , Win32-network   >= 0.1 && < 0.2
-                     , dns                       < 4.0
-                     , iproute         >= 1.7.0 && < 1.8
 
                        -- Used by the handshake protocol for now
                      , serialise       >=0.2 && <0.3
                        -- Remove once codec is defined locally
                      , typed-protocols-examples
-                       -- Drop dep once use of Data.Semigroup.Action is removed
-                     , cardano-prelude
 
   if os(windows)
-    build-depends:     Win32-network   <  0.2.0.0,
-                       Win32           >= 2.5.4.1 && <2.9
+    build-depends:     Win32           >= 2.5.4.1 && <2.9
 
   hs-source-dirs:      src
   default-language:    Haskell2010
   ghc-options:         -Wall
                        -Wcompat
-                       -Wincomplete-uni-patterns
-                       -Wincomplete-record-updates
-                       -Wpartial-fields
-                       -Widentities
-                       -Wredundant-constraints
+                       -Wincomplete-uni-patterns                            
+                       -Wincomplete-record-updates                          
+                       -Wpartial-fields                                     
+                       -Widentities                                         
+                       -Wredundant-constraints                              
```

Comment:
There's a lot of trailing whitespace in this file

Reply:
hmm, strange; I'll fix it.